src/modules/wallet/components/passphraseRenderer/index.js

```diff
@@ -130,24 +130,24 @@ class PassphraseRenderer extends React.Component {
     const otherIndex = indexes.find((index) => index !== selectedIndex);
     const shouldDisplayOptions = Object.values(chosenWords).length < 2;
 
-    this.setState({
-      ...this.state,
+    this.setState((state) => ({
+      state,
```

Comment:
This will result in a state object inside a state object. 
We need to spread the state

`...state,`

Reply:
Fixed